x-pack/plugins/cases/common/api/cases/comment/index.ts

```diff
@@ -233,7 +233,7 @@ export const CommentResponseTypePersistableStateRt = rt.intersection([
   }),
 ]);
 
-export const AllCommentsResponseRT = rt.array(CommentResponseRt);
+export const AllCommentsResponseRT = rt.array(CommentRt);
```

Comment:
I think this should be `CommentsRt` and the `CommentsRt` should be `CommentsFindResponseRt`. We can do it on another PR as we have the same issue for `Cases`.

Reply:
Also this seems like a duplicate of `AllCommentsResponseRt`. They differ in the last letter (capital `T`).